examples/tv-casting-app/tv-casting-common/include/CastingServer.h

```diff
@@ -58,17 +64,43 @@ class CastingServer
                                                 const chip::app::DataModel::DecodableList<chip::ClusterId> & responseList);
     static void OnDescriptorReadFailureResponse(void * context, CHIP_ERROR error);
 
-    CHIP_ERROR ContentLauncherLaunchURL(const char * contentUrl, const char * contentDisplayStr,
-                                        std::function<void(CHIP_ERROR)> launchURLResponseCallback);
-    static void OnContentLauncherSuccessResponse(
-        void * context, const chip::app::Clusters::ContentLauncher::Commands::LaunchResponse::DecodableType & response);
-    static void OnContentLauncherFailureResponse(void * context, CHIP_ERROR error);
+    [[deprecated("Use CastingServer.LaunchURL(..) instead")]] CHIP_ERROR
+    ContentLauncherLaunchURL(const char * contentUrl, const char * contentDisplayStr,
+                             std::function<void(CHIP_ERROR)> launchURLResponseCallback);
 
     chip::NodeId GetVideoPlayerNodeForFabricIndex(chip::FabricIndex fabricIndex);
     chip::FabricIndex GetVideoPlayerFabricIndexForNode(chip::NodeId nodeId);
     chip::FabricIndex CurrentFabricIndex() { return mTargetVideoPlayerInfo.GetFabricIndex(); }
     void SetDefaultFabricIndex();
 
+    CHIP_ERROR
+    LaunchURL(const char * contentUrl, const char * contentDisplayStr,
```

Comment:
You might want to prefix each command with the cluster name, otherwise it will become unwieldy. Maybe you were planning to do that in a subsequent re-factor?

eg.
ContentLauncherLaunchURL
ContentLauncherLaunchContent
LevelControlStep
MediaPlaybackPlay
...

Reply:
Makes sense. Done in commit#2.